modules/router-store/src/provide_router_store.ts

```diff
@@ -0,0 +1,63 @@
+import { Provider } from '@angular/core';
+import {
+  _createRouterConfig,
+  _ROUTER_CONFIG,
+  ROUTER_CONFIG,
+  RouterState,
+  StoreRouterConfig,
+} from './router_store_config';
+import {
+  FullRouterStateSerializer,
+  SerializedRouterStateSnapshot,
+} from './serializers/full_serializer';
+import { MinimalRouterStateSerializer } from './serializers/minimal_serializer';
+import {
+  BaseRouterStoreState,
+  RouterStateSerializer,
+} from './serializers/base';
+import { _initRouterStore } from './init_router_store';
+
+/**
+ * Provider function to connect the RouterModule to the StoreModule
+ *
+ * Usage:
+ *
+ * ```typescript
+ * @NgModule({
```

Comment:
This example should be using standalone `bootstrapApplication(AppComponent, { providers: [] })`

While you can use these with NgModules, we want to encourage usage with standalone APIs.

Reply:
Would you like me to assume there is a `provideStore` and such in the example? or just add `provideRouterStore` to the example?